### ISSUE TITLE
Anti-Virus went off

### ISSUE THREAD

**Issue by SparkyTheCat** [Vazity1337/Discord-Nitro-Generator#35]
I'm not sure to trust this, I downloaded the file, went to open it, and my Anti-Virus went off saying that file has a virus threat.  Should I not run the file, or am I ok? I just got this computer a month ago, I don't want a virus on it. Thank you

**Comment by codistiano** [comment]
Did you get any answer @SparkyTheCat ?

**Comment by eclipsek20** [comment]
If you do not know python then why bother using this? Tldr  ; your antivirus is shitty

**Comment by DaEpicSwag** [comment]
Lmao it’s the worlds simplest script. Here’s a guide:
These are the imports, something that adds a string value and a randomizer:
```python
import random, string
```
This is the input for how much you want to generate
```python
amount = int(input('Amount of nitro codes to generate: '))
value = 1
```
Lastly, here’s a loop to generate a syntax.
```python

while value <= amount:
    code = "https://discord.gift/" + ('').join(random.choices(string.ascii_letters + string.digits, k=16))
    f = open('Codes.txt', "a+")
    f.write(f'{code}\n')
    f.close()
    print(f'[GENERATED] {code}')
    value += 1
```

Also, don’t use this since there’s more of a chance to win the lottery then to get a code. Use this: https://github.com/Vedza/NitroSniper . It’s safe and I’ve used it and actually got nitro since I hosted it for 2 days on repl.it, if you want a tutorial on setting it up, I’ll make one, or just dm me on discord: SonicTheHedHog#0001

**Comment by JarnoXYZ** [comment]
this works, but i got terminated from discord after getting 1 nitro classic.
(the nitro sniper)

**Comment by DaEpicSwag** [comment]
True. This needs proxy support, since its obv that ur sniping from so many check requests.

**Comment by abdullahiftikhar-code** [comment]
> I'm not sure to trust this, I downloaded the file, went to open it, and my Anti-Virus went off saying that file has a virus threat. Should I not run the file, or am I ok? I just got this computer a month ago, I don't want a virus on it. Thank you

# Because it connects to the internet thats the reason it happen a lot of time its kinda weird
